src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,52 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, S> {
+    private T first;
+    private S second;
 
+    public Pair(T first, S second) {
```

Comment:
should constructor be private according to task?

Reply:
changed to ptivate.done